### ISSUE TITLE
"This extension may soon no longer be supported"

### ISSUE THREAD

**Issue by cglong** [plibither8/refined-hacker-news#135]
Chrome is reporting that Refined Hacker News may soon be disabled. Based on timing of this, I'm assuming the extension is written using Manifest v2, which would put the deadline at June 2025.

**Comment by isrbaral** [comment]
yup, seeing the same thing, appears this branch is abandoned, unfortunately there isn't anyone else who has published an official fork to the chrome webstore

**Comment by alexferrari88** [comment]
Please check my [fork](https://github.com/alexferrari88/refined-hacker-news). It works with Manifest V3. I will be submitting the extension to the Chrome Extensions store soon.

**Comment by eevmanu** [comment]
> Please check my [fork](https://github.com/alexferrari88/refined-hacker-news). It works with Manifest V3. I will be submitting the extension to the Chrome Extensions store soon.

hi @alexferrari88 I tried to install your fork, a couple of suggesting:

Do I need to run with a specific version of `node` and `npm`? I used these ones:

```sh
$ node --version
v22.19.0

$ npm --version
10.9.3
```

and when I run `npm install`, there were multiple changes on `package.json` and `package-lock.json` and when running `npm run build` there was an error related to webpack 🤔 .

So what I did was running `npm ci` instead of `npm install` to avoid any change in `package.json` and `package-lock.json` file and when running `npm run build`, everything looks good 👌🏼:

```sh
$ npm run build

> build
> webpack --mode=production


          background.js ⏤  3.05 kB (+3.05 kB)
          custom-css.js ⏤  821 B (+821 B)
               popup.js ⏤  3.91 kB (+3.91 kB)
 refined-hacker-news.js ⏤  61.1 kB (+61.1 kB)

/bin/sh: 1: stylus: not found
```

later when trying to `Load unpacked` the `dist/` folder there was an error related to not finding `refined-hacker-news.css` file:

```sh
$ grep -R 'refined' dist/ | grep -i css
./dist/manifest.json:      "css": ["refined-hacker-news.css"],

$ ls dist/ | grep -i css
custom-css.js
webext-base.css
```

so I need to change that part manually in `dist/manifest.json` and finally the installation of your fork works

As feedback I could say that the interaction by moving between comments is not so intuitive, probably because the rectangle hint around the comment does not appear 🤔 , don't know why.

Also the header looks a bit weird 🤔.

<img width="1784" height="187" alt="Image" src="https://github.com/user-attachments/assets/84589675-dbb9-4471-be35-911cb5263466" />

pd 1: writing the feedback here because the repo https://github.com/alexferrari88/refined-hacker-news doesn't have issues tab available
pd 2: i'm not so well versed into chrome extensions so take with a grain of salt this feedback please

**Comment by autarch** [comment]
You need to install stylus globally with `npm install -g stylus`, it seems. I did that and was able to get this to work.

**Comment by eevmanu** [comment]
Thx @autarch 

I didn't check the instructions for contributing, it was there.

https://github.com/plibither8/refined-hacker-news/blob/ee7ef6d55ae15f26a5bf4123d738b249883f7798/CONTRIBUTING.md#L41

After that missing CSS files appeared

```sh
$ ls dist/ | grep -i css
custom-css.js
popup.css
refined-hacker-news.css
webext-base.css
```

**Comment by orangejuiceext** [comment]
After many many years of using RHN, I've started to re-implement this extension, from the ground up.

https://github.com/OrangeJuiceExtension/OrangeJuice
